channels/offline-channel-js/src/Interfaces/IOfflineBatch.ts

```diff
@@ -36,7 +36,7 @@ export const enum eStorageType {
     IndexDb = 3
 }
 
-export const StorageType = createEnumStyle<typeof eStorageType>({
+export const StorageType = /* @__PURE__ */ createEnumStyle<typeof eStorageType>({
```

Comment:
Add parenthesis around the comment like the example, otherwise typescript doesn't handle this correctly